authorization/token/manager/token_manager.go

```diff
@@ -513,6 +514,26 @@ func (m *tokenManager) GenerateUnsignedUserAccessTokenForIdentity(ctx context.Co
 	return token, nil
 }
 
+// GenerateTransientUserAccessTokenForIdentity generates a transient user access token, an extremely short-lived token
+func (m *tokenManager) GenerateTransientUserAccessTokenForIdentity(ctx context.Context, identity repository.Identity) (*string, error) {
```

Comment:
what about using the term `action token` here?

Reply:
See my reply in Slack ;)